tiny-postgresql-event-store/src/main/kotlin/jp/veka/tables/Dto.kt

```diff
@@ -0,0 +1,91 @@
+package jp.veka.tables
+
+import jp.veka.converter.JsonEntityConverter
+import ru.quipy.domain.ActiveEventStreamReader
+import ru.quipy.domain.EventRecord
+import ru.quipy.domain.EventStreamReadIndex
+import ru.quipy.domain.Snapshot
+import ru.quipy.saga.SagaContext
+
+var entityConverter = JsonEntityConverter()
```

Comment:
everything should be injectable so that user can exchange the implementation on a fly. that can be done on PostgresClientEventStore level

Reply:
Still here